front/app/api/comments_by_topic/useCommentsByTopic.ts

```diff
@@ -0,0 +1,34 @@
+import { useQuery } from '@tanstack/react-query';
+import { CLErrors } from 'typings';
+import fetcher from 'utils/cl-react-query/fetcher';
+import {
+  ICommentsByTopic,
+  CommentsByTopicKeys,
+  ICommentsByTopicParams,
+} from './types';
+import commentsByTopicKeys from './keys';
+
+const fetchCommentsByTopic = (params: ICommentsByTopicParams) =>
+  fetcher<ICommentsByTopic>({
+    path: `/stats/comments_by_topic`,
+    action: 'get',
+    queryParams: params,
+  });
+
+const useCommentsByTopic = ({
+  enabled,
```

Comment:
Minor. Approach wise, I wonder if it isn't better to default to `enabled` true and only pass in `enabled` false when we want to turn it off 🤔 . More like make optional and default to true even for other scenarios in the PR. It is minor so I'll leave it to you

Reply:
I made the enabled prop mandatory on purpose just to make the mechanism more explicit and make sure it's always in place and whoever uses the hook is aware of it. Leaving as is for now :))